### PR TITLE
4.0 Doc Updates

### PR DESCRIPTION
**Pull request by jasonblais**
Includes 
 - Italian support
 - Removed Image settings in System Console
 - SQL query timeout support for PostgreSQL
 - Updated telemetry
 - Removed font and teammate name display account settings

### CODE REVIEW

File: source/administration/telemetry.rst [hunk 1]
```diff
@@ -40,15 +40,15 @@ Server Configuration Settings
...
...ool EnablePublicLink, enum DriverName, bool AmazonS3SSL, bool EnableFileAttachments, bool AmazonS3SignV2; **EmailSettings**: bool EnableSignUpWithEmail, bool EnableSignInWithEmail, bool EnableSignInWithUsername, bool RequireEmailVerification, bool SendEmailNotifications, enum ConnectionSecurity, bool SendPushNotifications, enum PushNotificationContents, bool EnableEmailBatching, bool SkipServerCertificateVerification; **RateLimitSettings**: bool EnableRateLimiter, bool VaryByRemoteAddr; **PrivacySettings**: bool ShowEmailAddress, bool ShowFullName; **GitLabSettings**: bool Enable; **GoogleSettings**: bool Enable; **Office365Settings**: bool Enable; **LdapSettings**: bool Enable, enum ConnectionSecurity, bool SkipCertificateVerification; **ComplianceSettings**: bool Enable, bool EnableDaily; **LocalizationSettings**: enum DefaultServerLocale, enum DefaultClientLocale, enum AvailableLocales; **SamlSettings**: bool Enable, bool Verify, bool Encrypt; **ClusterSettings**: bool Enable, bool UseIpAddress, bool UseExperimentalGossip, bool ReadOnlyConfig; **MetricsSettings**: bool Enable; **WebrtcSettings**: bool Enable, **AnnouncementSettings**: bool EnableBanner, bool AllowBannerDismissal
```
Comment (JeffSchering) [inline source/administration/telemetry.rst]: Can you double-check the SQL settings? This is what's in `diagnostics.go`:
``` go
SendDiagnostic(TRACK_CONFIG_SQL, map[string]interface{}{
		"driver_name":                 utils.Cfg.SqlSettings.DriverName,
		"trace":                       utils.Cfg.SqlSettings.Trace,
		"max_idle_conns":              utils.Cfg.SqlSettings.MaxIdleConns,
		"max_open_conns":              utils.Cfg.SqlSettings.MaxOpenConns,
		"data_source_replicas":        len(utils.Cfg.SqlSettings.DataSourceReplicas),
		"data_source_search_replicas": len(utils.Cfg.SqlSettings.DataSourceSearchReplicas),
		"query_timeout":               *utils.Cfg.SqlSettings.QueryTimeout,
	})
```

but this is what the **SqlSettings** are:

`**SqlSettings**: enum DriverName, bool Trace;`

Comment (JeffSchering) [inline source/administration/telemetry.rst]: Can you double-check **AnnouncementSettings** too?

``` go
SendDiagnostic(TRACK_CONFIG_ANNOUNCEMENT, map[string]interface{}{
		"enable_banner":               *utils.Cfg.AnnouncementSettings.EnableBanner,
		"isdefault_banner_color":      isDefault(*utils.Cfg.AnnouncementSettings.BannerColor, model.ANNOUNCEMENT_SETTINGS_DEFAULT_BANNER_COLOR),
		"isdefault_banner_text_color": isDefault(*utils.Cfg.AnnouncementSettings.BannerTextColor, model.ANNOUNCEMENT_SETTINGS_DEFAULT_BANNER_TEXT_COLOR),
		"allow_banner_dismissal":      *utils.Cfg.AnnouncementSettings.AllowBannerDismissal,
	})
```

Comment (jasonblais) [inline source/administration/telemetry.rst]: They were there, just in the wrong place..fixed, thanks!

Comment (jasonblais) [inline source/administration/telemetry.rst]: There's two settings in the `True/false (boolean)` section

**Pull request merged**